common/data/src/main/java/org/thingsboard/server/common/data/msg/TbMsgType.java

```diff
@@ -76,6 +76,7 @@ public enum TbMsgType {
     DEDUPLICATION_TIMEOUT_SELF_MSG(null, true),
     DELAY_TIMEOUT_SELF_MSG(null, true),
     MSG_COUNT_SELF_MSG(null, true),
+    DEVICE_STATE_STALE_ENTRIES_CLEANUP_MSG(null, true),
```

Comment:
Maybe add SELF:

```suggestion
    DEVICE_STATE_STALE_ENTRIES_CLEANUP_SELF_MSG(null, true),
```

Reply:
Renamed in https://github.com/thingsboard/thingsboard/pull/9030/commits/691a6b5d6a1cc0acd300eca5d9396e296207bf82